From 9c4cedacdce22e02407c7791d7c64cafddb62162 Mon Sep 17 00:00:00 2001
Message-Id: <patch-2.3-9c4cedacdce-20210921T224944Z-avarab@gmail.com>
In-Reply-To: <cover-0.3-00000000000-20210921T224944Z-avarab@gmail.com>
References: <cover-0.3-00000000000-20210921T224944Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 13 Sep 2021 13:55:39 +0200
Subject: [PATCH 2/3] Makefile: do one append in %.hcc rule
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change a rule added in added b503a2d515e (Makefile: emulate compile in
$(HCO) target better, 2019-09-25) to use one append, instead of ">"
followed by a ">>".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 5b09f67aab0..ce4063a031a 100644
--- a/Makefile
+++ b/Makefile
@@ -2914,8 +2914,10 @@ HCO = $(patsubst %.h,%.hco,$(CHK_HDRS))
 HCC = $(HCO:hco=hcc)
 
 %.hcc: %.h
-	@echo '#include "git-compat-util.h"' >$@
-	@echo '#include "$<"' >>$@
+	@{ \
+		echo '#include "git-compat-util.h"'; \
+		echo '#include "$<"'; \
+	} >$@
 
 $(HCO): %.hco: %.hcc FORCE
 	$(QUIET_HDR)$(CC) $(ALL_CFLAGS) -o /dev/null -c -xc $<
-- 
2.33.0.1098.gf02a64c1a2d

